dashboard/test/test_helper.rb

```diff
@@ -261,6 +261,10 @@ def assert_authentication_option(actual_option, expected_values)
     end
   end
 
+  def set_request_locale(locale)
```

Comment:
I thought we already had a way to mock the locale, but it looks like we were just forcing the request value to a certain locale:
https://github.com/code-dot-org/code-dot-org/blob/d16ce7f8772b6666c0914cf51654f4b1f5264cc3/dashboard/test/controllers/home_controller_test.rb#L185